overrides/scripts/_oreDict.zs

```diff
@@ -1330,32 +1340,28 @@ for oreDictEntry, items in miscDisabled {
 <ore:blockCoalCoke>.add(<metaitem:blockCoke>);
 
 //Ores
-<ore:oreIron>.add(<minecraft:iron_ore>);
-<ore:oreCoal>.add(<minecraft:coal_ore>);
-<ore:oreRedstone>.add(<minecraft:redstone_ore>);
-<ore:oreDiamond>.add(<minecraft:diamond_ore>);
-<ore:oreLapis>.add(<minecraft:lapis_ore>);
-<ore:oreGold>.add(<minecraft:gold_ore>);
-<ore:oreEmerald>.add(<minecraft:emerald_ore>);
-<ore:oreNetherQuartz>.add(<minecraft:quartz_ore>);
-
-<ore:blockAluminum>.add(<gregtech:meta_block_compressed_0:1>);
-
-<ore:blockCoalCoke>.add(<gregtech:meta_block_compressed_22:5>);
+mods.jei.JEI.removeAndHide(<minecraft:iron_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:coal_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:redstone_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:diamond_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:lapis_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:gold_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:emerald_ore>);
+mods.jei.JEI.removeAndHide(<minecraft:quartz_ore>);
```

Comment:
Do vanilla ores really have 0 cases where they show up? This might break stuff 
If this has been checked already, ignore this

Reply:
I remembered that they were allowed in GT Ore Proc, but I'll quickly check to see if they are anywhere.

Reply:
Yep no ways to get them except an oregen page which I assume is wrong.